pkg/services/servicebus/queue_bind.go

```diff
@@ -0,0 +1,31 @@
+package servicebus
+
+import (
+	"fmt"
+
+	"github.com/Azure/open-service-broker-azure/pkg/service"
+)
+
+func (qm *queueManager) Bind(
+	service.Instance,
+	service.BindingParameters,
+) (service.BindingDetails, error) {
+	return nil, nil
+}
+
+func (qm *queueManager) GetCredentials(
+	instance service.Instance,
+	_ service.Binding,
+) (service.Credentials, error) {
+	pdt := instance.Parent.Details.(*namespaceInstanceDetails)
+	dt := instance.Details.(*queueInstanceDetails)
+	return queueCredentials{
+		ConnectionString: string(pdt.ConnectionString),
+		PrimaryKey:       string(pdt.PrimaryKey),
+		QueueURL: fmt.Sprintf(
+			"https://%s.servicebus.windows.net/%s",
```

Comment:
Is it useful? I took a look at several SDKs. They need connection string + queue name, e.g. `queueClient = new QueueClient(ServiceBusConnectionString, QueueName);`. And there is no direct queue name in the credentials?